libs/api/domains/health-insurance/src/lib/soap/soapClient.ts

```diff
@@ -15,25 +15,39 @@ export class SoapClient {
     baseUrl: string,
     username: string,
     password: string,
+    clientID: string,
+    serviceID: string,
     functionName: string,
   ): Promise<Soap.Client | null> {
     logger.info('Starting create soapClient...')
+
```

Comment:
All the changes in this file seem to be almost identical to the ones in `libs/health-insurance/src/lib/soap/soapClient.ts`, can you join them in a single client?

Reply:
Done merged